### PR TITLE
Roman-converter-project

### PR DESCRIPTION
**Pull request by Sharifarezaie**
this is my Roman-converter-project
I added in this project styles.css, index.html, script.js

### CODE REVIEW

**Comment by Sharifarezaie** [comment]
@fatiabdulahi
Hi Fatema please check my project
thanks.

**Comment by Sharifarezaie** [comment]
@ahzamir
Hi sir i  hope you are well
you can check my project and share your opinion with us

**Comment by ahzamir** [comment]
@Sharifarezaie 

- You need to get your partner feedback first.

**Comment by fatiabdulahi** [comment]
Hello  dear Sharifa jan .I review your project and I think you are work
good .
I provide.

**Comment by Sharifarezaie** [comment]
Hello  dear teacher
I hope you are well
Can you review my project  thanks!

File: README.md [hunk 1]
```diff
@@ -1 +1,272 @@
...
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  updateUI();
+});
+
+convertButton.addEventListener('click', () => {
+  updateUI();
+});
+
+```
```
Comment (ahzamir) [inline README.md]: - Remove all of these codes from your README.md file as it is too much long

**Review by ahzamir — APPROVED** [review]
### Hi @Sharifarezaie 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Implement the final below changes then directly merge your branch 
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text with a screenshot.

- Well Done great design Implemented. 🎯🎉👍
image

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

WinkThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Roman Numeral Converter project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your repo has a professional README.md file
✔️ Your partner has reviewed your repo
✔️Your design is responsive

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by ahzamir** [comment]
Don't forget to deploy your project

**Pull request merged**